clusters/build-clusters/02_cluster/installation_params.md

```diff
@@ -0,0 +1,52 @@
+# Build02: installation parameters
+
+## [Configuring a GCP project](https://docs.openshift.com/container-platform/4.4/installing/installing_gcp/installing-gcp-account.html)
+
+Create a new project on GCP.
+
+### [Configuring DNS for GCP](https://docs.openshift.com/container-platform/4.4/installing/installing_gcp/installing-gcp-account.html#installation-gcp-dns_installing-gcp-account)
+
+* base-domain: It will be a part of the default hostname for routes.
+
+### [Creating a service account in GCP](https://docs.openshift.com/container-platform/4.4/installing/installing_gcp/installing-gcp-account.html#installation-gcp-service-account_installing-gcp-account)
+
+The service account key is required to create a cluster.
+
+GCP regions: us-east1 (Moncks Corner, South Carolina, USA)
+
+### pull-secret
+
+> oc get secret -n ci cluster-secrets-gcp  -o jsonpath='{.data.pull-secret}' | base64 -d
+
+(We use `cluster-secrets-aws` for `build01`.)
+
+## Customization
+
+### SSH key
+
+Install it. Using or not, it is the decision later.
```

Comment:
Can we rotate the thing down the lane? The problem here is that if someone who once had access to this key but shouldn't have it anymore can not be prevented from using it except by rotating it

Reply:
Considering the mistakes i made in the history, this is a very valid concern.

I guess this would be https://github.com/openshift/machine-config-operator/blob/master/docs/Update-SSHKeys.md

I have never tested it though.